graphene/__init__.py

```diff
@@ -1,38 +1,40 @@
 from .pyutils.version import get_version
-
-from .types import (
-    AbstractType,
-    ObjectType,
-    InputObjectType,
-    Interface,
-    Mutation,
-    Field,
-    InputField,
-    Schema,
-    Scalar,
-    String, ID, Int, Float, Boolean,
-    Date, DateTime, Time,
-    JSONString,
-    UUID,
-    List, NonNull,
-    Enum,
-    Argument,
-    Dynamic,
-    Union,
-    Context,
-    ResolveInfo
-)
-from .relay import (
-    Node,
-    is_node,
-    GlobalID,
-    ClientIDMutation,
-    Connection,
-    ConnectionField,
-    PageInfo
-)
-from .utils.resolve_only_args import resolve_only_args
+from .relay import ClientIDMutation
+from .relay import Connection
+from .relay import ConnectionField
+from .relay import GlobalID
+from .relay import is_node
+from .relay import Node
+from .relay import PageInfo
+from .types import AbstractType
+from .types import Argument
+from .types import Boolean
+from .types import Context
+from .types import Date
+from .types import DateTime
+from .types import Dynamic
+from .types import Enum
+from .types import Field
+from .types import Float
+from .types import ID
+from .types import InputField
+from .types import InputObjectType
+from .types import Int
+from .types import Interface
+from .types import JSONString
+from .types import List
+from .types import Mutation
+from .types import NonNull
+from .types import ObjectType
+from .types import ResolveInfo
+from .types import Scalar
+from .types import Schema
+from .types import String
+from .types import Time
+from .types import Union
+from .types import UUID
```

Comment:
I'm really not a fan of the imports being split up like this. How the imports were arranged is much clearer and easier to read.